Formula/m/mutt.rb

```diff
@@ -45,17 +47,19 @@ class Mutt < Formula
   uses_from_macos "krb5"
   uses_from_macos "zlib"
 
+  on_macos do
+    depends_on "gettext"
+    depends_on "libunistring"
+  end
+
   conflicts_with "tin", because: "both install mmdf.5 and mbox.5 man pages"
 
   def install
     user_in_mail_group = Etc.getgrnam("mail").mem.include?(ENV["USER"])
     effective_group = Etc.getgrgid(Process.egid).name
 
     args = %W[
-      --disable-dependency-tracking
       --disable-warnings
-      --prefix=#{prefix}
-      --enable-debug
```

Comment:
this was added per https://github.com/Homebrew/homebrew-core/commit/f474301d31ecb419219ccf64c6b4f021ac100469